themes/default/content/blog/slack-integration/index.md

```diff
@@ -0,0 +1,397 @@
+        Filters = new[]
+        {
+            PulumiService.WebhookFilters.StackCreated,
+            PulumiService.WebhookFilters.StackDeleted,
+            PulumiService.WebhookFilters.DeploymentSucceeded,
+            PulumiService.WebhookFilters.DeploymentFailed,
+        },
+    });
+
+    var stackWebhook = new PulumiService.Webhook("stackWebhook", new()
+    {
+        Active = true,
+        DisplayName = "#my-stack-activity-channel",
+        OrganizationName = "my-org",
+        ProjectName = "webhook-examples",
+        StackName = "dev",
+        PayloadUrl = "https://hooks.slack.com/services/...",
+        Format = PulumiService.WebhookFormat.Slack,
+    });
+
+    return new Dictionary<string, object?>
+    {
+        ["orgWebhookName"] = orgWebhook.Name,
+        ["stackWebhookName"] = stackWebhook.Name,
+    };
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language java %}}
+
+```java
+package generated_program;
+
+import com.pulumi.Context;
+import com.pulumi.Pulumi;
+import com.pulumi.core.Output;
+import com.pulumi.pulumiservice.Webhook;
+import com.pulumi.pulumiservice.WebhookArgs;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class App {
+    public static void main(String[] args) {
+        Pulumi.run(App::stack);
+    }
+
+    public static void stack(Context ctx) {
+        var orgWebhook = new Webhook("orgWebhook", WebhookArgs.builder()
+            .active(true)
+            .displayName("#my-org-activity-channel")
+            .organizationName("my-org")
+            .payloadUrl("https://hooks.slack.com/services/...")
+            .format("slack")
+            .filters(
+                "stack_created",
+                "stack_deleted",
+                "deployment_succeeded",
+                "deployment_failed")
+            .build());
+
+        var stackWebhook = new Webhook("stackWebhook", WebhookArgs.builder()
+            .active(true)
+            .displayName("#my-stack-activity-channel")
+            .organizationName("my-org")
+            .projectName("webhook-examples")
+            .stackName("dev")
+            .payloadUrl("https://hooks.slack.com/services/...")
+            .format("slack")
+            .build());
+
+        ctx.export("orgWebhookName", orgWebhook.name());
+        ctx.export("stackWebhookName", stackWebhook.name());
+    }
+}
+```
+
+{{% /chooser %}}
+
+## Pulumi Deployment Notification Events
+
+Pulumi Deployments is infrastructure deployments as a managed service. With Pulumi Deployments, you can run a Pulumi infrastructure as code action (a preview, update, destroy or refresh) inside Pulumi’s managed service. Pulumi provides scalability, observability and security for deployments. Both Slack-formatted and generic JSON webhooks in Pulumi Cloud now send notifications events on Pulumi Deployments statuses: when a deployment is queued, started, succeeds and fails.
```

Comment:
I would just link the first sentence from the deploy docs:

 Pulumi Deployments is a fully managed platform for managing infrastructure at scale.

Also, might just be useful to show a screenshot here of a deploy-specific slack notification just to break up the text. Maybe a queued notif?